src/elements.js

```diff
@@ -220,6 +220,39 @@ _.register({
 		subtree: true
 	},
 
+	"select[multiple]": {
+		extend: "select",
+		selector: "select[multiple]",
+		getValue: element => {
+			const selected = element.selectedOptions;
+			const value = [];
+
+			if (selected.length) {
```

Comment:
I think just `Array.from(element.selectedOptions).map(option => option.value)` should work, but I haven't tested it.

Reply:
Yes, it works. Thank you. This is much much clearer.